builtin/providers/aws/resource_aws_elb.go

```diff
@@ -438,6 +467,22 @@ func resourceAwsElbUpdate(d *schema.ResourceData, meta interface{}) error {
 			},
 		}
 
+		logs := d.Get("access_logs").(*schema.Set).List()
+		if len(logs) > 0 {
```

Comment:
Right. Unfortunately, this isn't great because it will silently discard anything but the first occurrence. IMHO, we should return an error back to the user if there is > 1 block so there isn't any ambiguity.